src/testUtils.ts

```diff
@@ -196,11 +233,18 @@ function expandFilePathInOutput(output: string, cwd: string): string {
 }
 
 /**
+ * Function type for getting the target arguments.
+ *
+ * @param testconfig Configuration for the Go extension.
+ */
+type argsHandleFunc = (testconfig: TestConfig) => Thenable<Array<string>>;
+
+/**
  * Get the test target arguments.
  *
  * @param testconfig Configuration for the Go extension.
  */
-function targetArgs(testconfig: TestConfig): Thenable<Array<string>> {
+function testTargetArgs(testconfig: TestConfig): Thenable<Array<string>> {
 	if (testconfig.functions) {
 		return Promise.resolve(['-run', util.format('^%s$', testconfig.functions.join('|'))]);
```

Comment:
Instead of having a separate `benchmarkTargetArgs`, you can just use `-bench` here instead of `-run` if `testconfig.isBenchmark` is `true`